qodana.yaml

```diff
@@ -1,5 +1,5 @@
 version: 1.0
-failThreshold: 64
+failThreshold: 0
```

Comment:
Do you know if this also picks up on info comments from Qodana? Just a question.

Reply:
No, It counts 'problems' as in notices, warnings, and errors